aiokafka/structs.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import annotations
```

Comment:
There is a couple of linting errors
```suggestion
from __future__ import annotations

```